context/app/static/js/stores/useSearchDatasetTutorialStore.js

```diff
@@ -1,22 +1,46 @@
 import create from 'zustand';
+import ReactGA from 'react-ga';
 
 const localStorageKey = 'has_exited_dataset_search_tutorial';
 
-const useSearchDatasetTutorialStore = create((set) => ({
+const useSearchDatasetTutorialStore = create((set, get) => ({
   runSearchDatasetTutorial: false,
-  setRunSearchDatasetTutorial: (val) => set({ runSearchDatasetTutorial: val }),
+  setRunSearchDatasetTutorial: (val) => {
+    ReactGA.event({
+      category: 'Dataset Search Tutorial',
+      action: 'Clicked start button',
+    });
+    set({ runSearchDatasetTutorial: val });
+  },
   searchDatasetTutorialStep: 0,
-  incrementSearchDatasetTutorialStep: () =>
+  incrementSearchDatasetTutorialStep: () => {
+    ReactGA.event({
+      category: 'Dataset Search Tutorial',
+      action: 'Clicked next step',
+      value: get().searchDatasetTutorialStep,
+    });
```

Comment:
I might add a helper function so you could use less boilerplate:
```suggestion
    recordEvent('Clicked next step', get().searchDatasetTutorialStep);
```

Reply:
or, even more abbreviated:
```diff
- const useSearchDatasetTutorialStore = create((set, get) => ({
+ const useSearchDatasetTutorialStore = create((set, get) => {
+   function recordEvent(action) {
+      ReactGA.event({
+        category: 'Dataset Search Tutorial',
+        action,
+        value: get().searchDatasetTutorialStep,
+      });
+    }
+   return {
```